lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -230,11 +230,13 @@ sub sync_tests {
             log_info('Finished to rsync tests', channels => 'autoinst');
             last;
         }
-        elsif ($remaining_tries > 1 && $result eq 'exit code 24') {
+        elsif ($remaining_tries > 1 && ($result eq 'exit code 24' || $result eq 'exit code 23')) {
             log_info("Rsync failed due to vanished source files ($result), trying again", channels => 'autoinst');
```

Comment:
Logging this message in case of `exit code 23` is misleading. I would rather log `"Partial transfer due to error ($result)", trying again"` in this case.

Reply:
+1

Reply:
oh, do you mean just replace the string or add another, explicit log line for the different exit code? I would keep a single line as you already have the exit code for distinction